src/components/MapControls/MenuToggler/MenuToggler.js

```diff
@@ -11,21 +12,32 @@ const useStyles = makeStyles({
   displayMenuToggler: { padding: "8px" },
 });
 
-function MenuToggler() {
+function MenuToggler({ children, title, onClick, className, ...props }) {
   const classes = useStyles();
   const { t } = useTranslation();
   const displayMenu = useSelector((state) => state.app.displayMenu);
   const dispatch = useDispatch();
   return (
     <MapButton
-      className={`wkp-display-menu-toggler ${classes.displayMenuToggler}`}
-      onClick={() => dispatch(setDisplayMenu(!displayMenu))}
-      title={t("Menü")}
+      className={
+        className ?? `wkp-display-menu-toggler ${classes.displayMenuToggler}`
+      }
+      onClick={() => onClick?.() ?? dispatch(setDisplayMenu(!displayMenu))}
```

Comment:
Here you test the value returned by the onClick. Are you sure this is what you want ?